blazor/rich-text-editor/data-binding.md

```diff
@@ -9,7 +9,7 @@ documentation: ug
 
 # Value Binding in Blazor RichTextEditor
 
-You can bind the [Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.SfRichTextEditor.html#Syncfusion_Blazor_RichTextEditor_SfRichTextEditor_Value) to the editor component, by using the `@bind-Value` attribute. If component value has been changed, it will affect all the places where the variable is bound for the bind-value attribute.
+You can bind the [Value](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.SfRichTextEditor.html#Syncfusion_Blazor_RichTextEditor_SfRichTextEditor_Value) to the editor component, by using the `@bind-Value` attribute and it supports string type. If component value has been changed, it will affect all the places where you bind the variable for the **bind-value** attribute.
```

Comment:
**bind-value**  - is it correct and cases?

Reply:
Yes sir. Correct case

<img width="401" alt="image" src="https://user-images.githubusercontent.com/93525217/201015334-77d96956-d039-4f81-a925-92f65524799a.png">